doc/manuals/devel/httpHeaders.md

```diff
@@ -0,0 +1,329 @@
+	
+    curl ... -H 'Accept: application/json'
+		 
+On this request we'll get the response data in well defined JSON format.
+
+The API response payloads in this specification are based on `application/json` and  (for attribute value 
+type operation) `text/plain` MIME types. Clients issuing HTTP requests with accept types different 
+than those will get a `406 Not Acceptable` error.
+
+[Top](#top)
+
+## 2. Content-Length
+
+This header is used by both requests and responses.
+Content-length HTTP header is a mandatory header in Orion responses.
+It defines the length of the request and response body in bytes.
+Orion Context Broker expects always a Content-Length header in all client requests, otherwise the client will receive a `411 Length Required` response. 
+This is due to the way the underlying HTTP library (libmicrohttpd) works.
+	
+    Content-Length: 34	
+
+[Top](#top)
+ 
+## 3. Content-Type 
+
+It is used to specify the MIME type of the request or response. Typically its value is `application/json` (for JSON MIME type) although some operations in the NGSIv2 API also allow `text/plain`.
+
+    curl ... -H 'Content-Type: application/json'
+
+This request will send data in well defined JSON format.
+
+[Top](#top)
+ 
+## 4. Connection
+ 
+ This header is used by both requests and responses.
+ Orion is not currently doing anything with this header (apart from checking if its value is `close` to print a log trace).
+	
+    Connection: Keep-Alive
+    	
+[Top](#top)	
+  
+## 5. Expect
+
+Expect HTTP header is used in requests.
+It Indicates that behaviour of particular server are required by the client.
+It is typically used when sending a large request body. We expect the server to return back `100 Continue` HTTP status if it can handle the request, or `417 Expectation Failed` if not.
+
+    Expect: 100-continue
+	
+[Top](#top)	
+	
+## 6. Host
+
+This HTTP header is used in Orion responses.
+Orion includes this header in outgoing request (notifications and forwarded updates/queries).This header provides the port number of notification receiver. 	
+	
+    Host: localhost:1028
+		
+[Top](#top)
+
+## 7. Origin
+
+Origin is used in requests.
+Its operation is related with CORS, see specific documentation at [../user/cors.md](https://github.com/telefonicaid/fiware-orion/blob/master/doc/manuals/user/cors.md).
+	
+    Origin: http://www.example-social-network.com
+		
+[Top](#top)	
+  
+## 8. User-Agent
+
+User-Agent HTTP header is used in notifications and forwarded requests.
+It describes which version of Orion we are using along with its transfer library.
+It gives the string that identifies the user agent.
+
+    User-Agent: orion/2.2.0-next libcurl/7.29.0     (in notifications and forwarded requests)
+
+[Top](#top)		
+		
+## 9. X-Forwarded-For
+
+It is a request HTTP header in Orion.
+Source IP of the HTTP request associated to the transaction, except if the request includes X-Forwarded-For header which overrides the former IP.
```

Comment:
```suggestion
X-Forwarded-For header overrides the original IP of the HTTP request as source of the transaction.
```

And a similar adaptation in the other place in which this text appear, i.e. X-Real-IP.

Reply:
Fixed in a427bd9